### ISSUE TITLE
wav2vec2.0 cer is significant high than wav2letter in the same chinese training dataset

### ISSUE THREAD

**Issue by luweishuang** [pytorch/fairseq#2552]
I used wav2vec2.0 in chinese to do asr task and I used "LexiconFreeDecoder" in  facebookresearch/wav2letter instead of "LexiconDecoder" to do beam-search decoder.  I compared the cer in wav2vec2.0 and streaming_convnets of wavletter using the same training dataset and found the latter have a significant advantage in cer. Is this a normal situation for wav2vec2.0? It seem like wav2vec2.0 features is not very suitable for asr task when compared to fbank feature and mfsc feature.  I used 400 hours unsupervised audios to finetune  "Wav2Vec 2.0 Large (LV-60)" model to get chinese version wav2Vev2.0-Large and then using 200h audios to finetune this chinese version model with ctc loss. Finally I used LexiconFreeDecode to decoder and calculate cer. Is there any wrong with my running steps?
